vllm/model_executor/custom_op.py

```diff
@@ -130,13 +130,11 @@ def default_on() -> bool:
         On by default if PyTorch Inductor is not used.
         Specifying 'all' or 'none' in custom_op takes precedence.
```

Comment:
```suggestion
       Behavior controlled by `CompilationConfig.custom_ops`: On by default if 'all', off by default if 'none'.
       When PyTorch Inductor is used, 'none' is the default value, otherwise 'all'.
```